app/src/app/components/header/NavMenu.tsx

```diff
@@ -0,0 +1,71 @@
+import { hasPacketRunPermission, hasUserManagePermission } from "../../../lib/auth/hasPermission";
+import { cn } from "../../../lib/cn";
+import { UserState } from "../providers/types/UserTypes";
+import { DropdownMenu, DropdownMenuTrigger, DropdownMenuContent, DropdownMenuItem } from "../Base/DropdownMenu";
+import { Menu } from "lucide-react";
+import { NavLink } from "react-router-dom";
+import { Button, buttonVariants } from "../Base/Button";
+
+export const NavItems: { [key: string]: string } = {
+  runner: "Runner",
+  "manage-roles": "Manage Access"
+  // "run-workflow": "Workflow Runner",
+  // documentation: "Documentation",
+  // accessibility: "Accessibility",
```

Comment:
I think we can probably retire our expectations of having a Workflow Runner any time soon, and Documentation is going to be handled elsewhere. Accessibility page remains an aspiration.